app/controllers/authors_controller.rb

```diff
@@ -11,5 +13,7 @@ def mine
 
   def set_author
     @author = Author.find(params[:id])
+    @posts = Node.get_nodes_readable_by(current_user) { |q| where("author_id = #{author.id}") }
```

Comment:
[ ] Currently broken. The `get_nodes_readable_by` method has been removed from `Node`.

Reply:
Fix submitted https://github.com/fi-tools/cf-forum/pull/19/commits/b3572b6f0295cc18d29dc6d9e91254623ba6fc9b